english/explore-outside/call-scotty/call-scotty.md

```diff
@@ -1,3 +1,5 @@
-You feel inside your pockets and, suprisingly, you find a Starfleet communicator. 
+You feel inside your pockets and, suprisingly, you find a Starfleet communicator.
 
-You flip it open, press the send button and say "Beam me up, Scotty!"
\ No newline at end of file
+You flip it open, press the send button and say "Beam me up, Scotty!"
+
+As you beam up, you see a light in the sky of a ship with a rocket heading towards it, you just hope its not the one you are beaming up to...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.